UiPath.Script.Activities.Design/DesignerMetadata.cs

```diff
@@ -12,13 +14,27 @@ public class DesignerMetadata : IRegisterMetadata
         public void Register()
         {
             var builder = new AttributeTableBuilder();
+
+            var categoryAttribute = new CategoryAttribute("App Scripting");
             var autoHotKeyType = typeof(ExecuteAutoHotKey);
```

Comment:
I think we should change the definition to "RunAutoHotKeyScript", for naming consistency